bench/wolfssl/optimized/ssl.c.ll

```diff
@@ -4673,14 +4673,17 @@ do.end195.thread:                                 ; preds = %if.then184, %if.the
   call void @FreeDecodedCert(ptr noundef nonnull %cert) #20
   br label %do.end206
 
-if.then202:                                       ; preds = %if.end127
+do.end195:                                        ; preds = %if.end127
   call void @FreeDecodedCert(ptr noundef nonnull %cert) #20
+  br i1 %cmp104.not, label %do.end206, label %if.then202
```

Comment:
Regression.

Reply:
Dominate chain:
if.end100 -> if.then106 -> if.end118 -> if.end127 -> do.end195
%cmp104.not is always false here.